code/modules/jobs/job_types/_job.dm

```diff
@@ -155,6 +155,16 @@
 	if(!ishuman(spawned))
 		return
 
+	var/mob/living/carbon/human/spawned_human = spawned
+
+	// Look for PDA in belt or left pocket to update its ringtone based on our client's prefs.
+	var/obj/item/modular_computer/tablet/pda/pda = spawned_human.belt
```

Comment:
``/datum/outfit/job/post_equip`` already does some PDA stuff and does it in a pretty nice way that doesn't have hardcoded paths like you have here, do you think you can put it there instead?

Reply:
I figured out a way to do it,